### ISSUE TITLE
RFE: Exclude some files/extensions/directories for `changedfiles`

### ISSUE THREAD

**Issue by omajid** [rpminspect/rpminspect#351]
I work in a language/stack that doesn't reproduce builds. Building the exact same sources produces a different set of files.

rpminspect complains about this. For example:

```
8) File /usr/lib64/dotnet/sdk/5.0.104/Extensions/Microsoft.TestPlatform.TestHostRuntimeProvider.dll changed content on x86_64.  Please verify this change was deliberate for a non-rebased build.
```

I couldn't find anything in the docs that covers how to exclude some files to avoid repeat of this warning on every build.

**Comment by dcantrell** [comment]
Using the example above with two dotnet5.0 builds, I have been unable to get the same result from the changedfiles inspection.  That may be due to commit 899144a7315e1208d6675512ca8ad75dc66eefe3 which adjusted how changedfiles works.

Nonetheless, I have added per-inspection ignore glob(7) support.  So, assuming the above file shows up as a false-positive in the changedfiles inspection you can tell rpminspect to ignore it just for changedfiles by adding this to your rpminspect.yaml file dist-git:

```
changedfiles:
    ignore:
        - /usr/lib64/dotnet/sdk/5.0.104/Extensions/Microsoft.TestPlatform.TestHostRuntimeProvider.dll
```

The entries listed here can follow glob(7) syntax.

**Comment by omajid** [comment]
Thanks!
